src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/BitcoinActivationRule.cs

```diff
@@ -1,17 +1,15 @@
-﻿using System.Threading.Tasks;
-using Microsoft.Extensions.Logging;
+﻿using Microsoft.Extensions.Logging;
 using NBitcoin;
 using Stratis.Bitcoin.Consensus.Rules;
 using Stratis.Bitcoin.Networks;
-using Stratis.Bitcoin.Utilities;
 
 namespace Stratis.Bitcoin.Features.Consensus.Rules.CommonRules
 {
     /// <summary>
-    /// Check that a <see cref="BitcoinMain"/> network block has the correct version according to the defined active deployments.
+    /// Check that a <see cref="BitcoinMain" /> network block has the correct version according to the defined active deployments.
     /// </summary>
-    [HeaderValidationRule]
-    public class BitcoinActivationRule : ConsensusRule
+    /// <remarks>This is header validation rule.</remarks>
```

Comment:
Why not have four base classes so that you do not need this remarks comment.

Reply:
done